prod-docker-compose.yml

```diff
@@ -48,12 +48,13 @@ services:
       - CACHE_DIR=/data/cache
       - ORIGIN=http://origin/
     extra_hosts:
-      # change it so private IPs are not hard coded 
-      - "origin:10.0.156.95" 
-      - "b.com:10.0.156.95" 
-      - "a.com:10.0.156.95" 
+      # change it so private IPs are not hard coded
+      - "origin:10.0.156.95"
+      - "b.com:10.0.156.95"
+      - "a.com:10.0.156.95"
     volumes:
       - /mnt/image-cache-part/cache:/data/cache
+      - /log/:/log/
```

Comment:
Can we store logs in `/mnt/image-cache-part/log`? I believe that unix has no such directory as `/log` by itself (and it is rather a bad practise to put some new folders under root directory).